utility/src/main/java/org/oppia/android/util/logging/firebase/FirebaseEventLogger.kt

```diff
@@ -60,5 +80,8 @@ class FirebaseEventLogger private constructor(
      */
     fun create(): EventLogger =
       FirebaseEventLogger(firebaseAnalytics, networkConnectionUtil, eventBundleCreator)
+
+    fun createPerformanceMetricEventLogger(): PerformanceMetricsEventLogger =
```

Comment:
Please add a KDoc for this. Also, any new tests for this method?

Reply:
added kdoc.